extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/common/GooglePhotosImportUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.datatransferproject.datatransfer.google.common;
+
+import com.google.common.base.Strings;
+
+public class GooglePhotosImportUtils {
```

Comment:
This seems like some work that might already be present on the model itself: https://github.com/google/data-transfer-project/blob/master/portability-types-common/src/main/java/org/datatransferproject/types/common/models/videos/VideoModel.java#L126-L141

Can we add these utility classes on the model instead of adding them here?

Reply:
Thanks for bringing it up. I see that there's `TransmogrificationConfig` along with the `transmogrify` method in the model which is made to solve that exact problem. However, I lack the specific knowledge to implement the correct config for Google Photos. Perhaps it could be done in a dedicated ticket.